ui/src/Components/Dashboard/Dashboard.js

```diff
@@ -1,10 +1,5 @@
-import Table from "./Table/Table"
+import Table from "./Table/Table";
 
 export default function Dashboard({ featureFlags, headers }) {
-  return (
-          <Table
-            featureFlags={featureFlags}
-            headers={headers}
-          /> 
-        )}
-      
\ No newline at end of file
+  return <Table featureFlags={featureFlags} headers={headers} />;
```

Comment:
Same is followed everywhere. This is not a change from this PR. It looks like it because i ran perttier. Let me know if you still need me to change it because all components follow the same

Reply:
Ok, we can change this to arrow function